src/Components/Filter/ProductFilter.js

```diff
@@ -2,62 +2,84 @@ import "./ProductFilter.css";
+                <input type="range" min="0" max="100" /><br/><br/>
             </li>
 
-            <h3 clasName="filter-items-heading">Sort</h3>
-            <li clasName="sidebarItem">
-                <input type="radio" clasName="filter-item" value="price low to high" name="sort-btn"/>Price Low to high
+            <h3 
+            className="filter-items-heading">Sort</h3>
+            <li 
+            className="sidebarItem">
+                <input type="radio" 
+                className="filter-item"  name="sort-btn"/>Price Low to high
             </li>
-            <li clasName="sidebarItem">
-                <input type="radio" value="price low to high" name="sort-btn"/>Price High to Low
+            <li 
+            className="sidebarItem">
+                <input type="radio"  name="sort-btn"/>Price High to Low
             </li>
 
-            <h3 clasName="filter-items-heading">Categories</h3>
-            <li clasName="sidebarItem">
-                <input type="checkbox" value="Men" name="categories-btn"/>Men
+            <h3 
+            className="filter-items-heading">Categories</h3>
+            <li 
+            className="sidebarItem">
+                <input type="checkbox"  name="categories-btn"/>Men
             </li>
-            <li clasName="sidebarItem">
-                <input type="checkbox" value="Woman" name="categories-btn"/>Woman
+            <li 
+            className="sidebarItem">
+                <input type="checkbox" name="categories-btn"/>Woman
             </li>
 
-            <h3 clasName="filter-items-heading">Brand</h3>
-            <li clasName="sidebarItem">
-                <input type="checkbox" value="Reebok" name="Brand-btn"/>Reebok
+            <h3 
+            className="filter-items-heading">Brand</h3>
+            <li 
+            className="sidebarItem">
+                <input type="checkbox"  name="Brand-btn"/>Reebok
             </li>
 
-            <li clasName="sidebarItem">
-                <input type="checkbox" value="Lotto" name="Brand-btn"/>Lotto
+            <li 
+            className="sidebarItem">
+                <input type="checkbox"  name="Brand-btn"/>Lotto
             </li>
-            <li clasName="sidebarItem">
-                <input type="checkbox" value="Fila" name="Brand-btn"/>Fila
+            <li 
+            className="sidebarItem">
+                <input type="checkbox"  name="Brand-btn"/>Fila
             </li>
-            <li clasName="sidebarItem">
-                <input type="checkbox" value="Hummel" name="Brand-btn"/>Hummel
+            <li 
+            className="sidebarItem">
+                <input type="checkbox"  name="Brand-btn"/>Hummel
             </li>
-            <li clasName="sidebarItem">
-                <input type="checkbox" value="Asics" name="Brand-btn"/>Asics
+            <li 
+            className="sidebarItem">
+                <input type="checkbox"  name="Brand-btn"/>Asics
             </li>
```

Comment:
For brands you can create an array and map over it instead of repeating all the html code for it  explicitly. Just a suggestion.